owlbot.py

```diff
@@ -27,6 +27,7 @@
         ".github/release-please.yml",
         ".github/workflows/auto-release.yaml",
         ".github/workflows/samples.yaml",
+        ".github/sync-repo-settings.yaml",
```

Comment:
sync repo settings will not be edited by owlbot. It doesn't have to be in the exclusion list.